buddyboss/buddyboss-platform
# PROD-897 Fix/1735

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1735 .

### How to test the changes in this Pull Request:

1. Subscribe to any discussions
2. Go to your profile page.
3. Navigate to Forums tab.
4. Now, got to Subscriptions and try to click any avatars from the list.
5. See the error

### Proof Screenshots or Video
https://www.loom.com/share/e0ffec8f95b34e8aa8eaaaa1de100288



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
